cmd/kube-controller-manager/app/plugins.go

```diff
@@ -43,7 +43,22 @@ func ProbeRecyclableVolumePlugins() []volume.VolumePlugin {
 	// The list of plugins to probe is decided by the kubelet binary, not
 	// by dynamic linking or other "magic".  Plugins will be analyzed and
 	// initialized later.
-	allPlugins = append(allPlugins, host_path.ProbeVolumePlugins()...)
-	allPlugins = append(allPlugins, nfs.ProbeVolumePlugins()...)
+	allPlugins = append(allPlugins, host_path.ProbeVolumePlugins(defaultRecycler(60))...)
```

Comment:
First: I really hate that the signature is different for  some of these Probes.  Can we find a way to not muck with the function args?

Second: since these numbers are totally magic, what's the value of injecting them?  I know this is trying to move the duplication out, but this feels TOO far out.  Why not just encapsulate defaultRecycler() into the volume util lib?

Third: these magic numbers won't last long before they need to be tweaked.  They are finger-in-the-wind right now.  We'd better plan to make the aborting of a scrub instead be based on lack-of-progress or scale time by size of volume or something.